pkg/maintainer/wallet/redemptions.go

```diff
@@ -1,23 +1,22 @@
-package coordinator
+package wallet
 
 import (
 	"errors"
 	"fmt"
+	"github.com/keep-network/keep-core/internal/hexutils"
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/tbtc"
 	"math/big"
 	"sort"
 	"time"
-
-	"github.com/keep-network/keep-core/pkg/bitcoin"
-	"github.com/keep-network/keep-core/pkg/tbtc"
-
-	"github.com/keep-network/keep-core/internal/hexutils"
 )
 
-type redemptionEntry struct {
-	walletPublicKeyHash  [20]byte
-	redemptionKey        string
-	redeemerOutputScript bitcoin.Script
-	requestedAt          time.Time
+// RedemptionRequest represents a redemption request.
+type RedemptionRequest struct {
```

Comment:
This type is only used in the `redemptions` package internally. Should we export it?

Reply:
I wanted to have a consistent public API of the `pkg/maintainer/wallet`. We export `Deposit` from `deposit_sweep.go` so it makes sense to export a representation of the redemption request as well.